docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>Is this screenshot out of date now, missing the "Draw" button?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Updating now.</p>